src/styles/homeScreenStyle.js

```diff
@@ -87,6 +90,14 @@ const styles = StyleSheet.create({
     fontSize: 14,
     color: '#2e78b7',
   },
+
+  imageBackground: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: null,
+    height: null,
```

Comment:
I think this property could not be null or remove them

Reply:
Changes applied